react-state/my-app/src/ToggleButton.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import './ToggleButton.css';
+
+function ToggleButton({ color, text }) {
+  let [isClicked, setClick] = useState(false);
+
+  console.log('useState', isClicked);
+
+  function handleClick() {
+    console.log('before setter', isClicked);
+    setClick(() => !isClicked);
+    console.log('after setter', isClicked);
+  }
+
+  return isClicked ? (
+    <button className={color} onClick={handleClick}>
+      {text}
+    </button>
+  ) : (
+    <button className="white" onClick={handleClick}>
+      {text}
+    </button>
+  );
+}
```

Comment:
Works, but since the only thing that's changing here is the `className`, we don't need to do a whole ternary for 2 different `button`s. We can just do it for the `className`.

Example 1:
```
return (
  <button className={isClicked ? color : 'white'} onClick={handleClick}>
        {text}
  </button>
  )
```

Example 2:
```
const bgkColor = isClicked ? color : 'white';
return (
  <button className={bgkColor} onClick={handleClick}>
        {text}
  </button>
  )
  ```